src/AspNetCore/Inc/inprocessapplication.h

```diff
@@ -53,8 +54,14 @@ class IN_PROCESS_APPLICATION : public APPLICATION
 
     HRESULT
     LoadManagedApplication(
-            VOID
-        );
+        VOID
+    );
+    
+    HRESULT
+    OnClientDisconnect(
+        IHttpContext* pHttpContext,
+        BOOL fClientDisconnected
```

Comment:
I don't think we need this fClientDisconnected, since this value is not flowed to managed layer.